etterlattemaler/src/main/kotlin/no/nav/pensjon/etterlatte/maler/vedlegg/omstillingsstoenad/InformasjonOmOmstillingsstoenad.kt

```diff
@@ -417,44 +417,106 @@ private fun OutlineOnlyScope<LangBokmalNynorskEnglish, InformasjonOmOmstillingss
+                Nynorsk to "Les meir på ${Constants.SKATTEETATEN_KILDESKATTPENSJON_URL} eller kontakt Skatteetaten på telefon ${Constants.SKATTEETATEN_KONTAKTTELEFON_MED_LANDKODE} frå utlandet.",
+                English to "Read more at ${Constants.SKATTEETATEN_KILDESKATTPENSJON_URL} or contact the Norwegian Tax Administration by phone at ${Constants.SKATTEETATEN_KONTAKTTELEFON_MED_LANDKODE} from abroad."
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "Omstillingsstønaden er pensjonsgivende inntekt. Den gir ikke opptjening av feriepenger.",
+                Nynorsk to "Omstillingsstønaden er pensjonsgivande inntekt. Den gir ikkje opptening av feriepengar.",
+                English to "Adjustment allowance are considered pensionable income. They do not earn you holiday pay.",
+            )
+        }
+    }.orShow {
+        paragraph {
+            text(
+                Bokmal to "Omstillingsstønaden er skattepliktig. Du trenger ikke levere skattekortet til Nav " +
+                        "fordi skatteopplysningene dine sendes elektronisk fra Skatteetaten.",
+                Nynorsk to "Omstillingsstønaden er skattepliktig. Du treng ikkje levere skattekortet til Nav, " +
+                        "då skatteopplysningane dine blir sende elektronisk frå Skatteetaten.",
+                English to "Adjustment allowance are taxable. You do not need to submit your tax deduction " +
+                        "card to Nav because your tax information is sent to Nav electronically from the " +
+                        "Norwegian Tax Administration.",
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "Endring av skattekort gjøres enklest på Skatteetatens nettsider www.skatteetaten.no. " +
+                        "Har du spørsmål kan du ringe Skatteetaten på telefon " + Constants.KONTAKTTELEFON_SKATT + ". " +
+                        "Fra utlandet ringer du " + Constants.Utland.KONTAKTTELEFON_SKATT + ".",
+                Nynorsk to "Skattekortet endrar du enklast frå nettsidene til Skatteetaten, www.skatteetaten.no. " +
+                        "Viss du har spørsmål, kan du ringje Skatteetaten på telefon " + Constants.KONTAKTTELEFON_SKATT + ". " +
+                        "Frå utlandet ringjer du " + Constants.Utland.KONTAKTTELEFON_SKATT + ".",
+                English to "The easiest way to change your tax deduction card is done on the Tax Administration's " +
+                        "website: www.skatteetaten.no. If you have any questions, please call the Tax Administration " +
+                        "by phone: " + Constants.KONTAKTTELEFON_SKATT + ". For calls from abroad: " +
+                        Constants.Utland.KONTAKTTELEFON_SKATT + ".",
+            )
+        }
+        paragraph {
```

Comment:
Denne kan legges etter .orShow da den også er felles for bosattUtland. Så kan paragraph på linje 480 fjernes.